app/lib/features/news/providers/news_providers.dart

```diff
@@ -5,7 +5,7 @@ import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
 import 'package:riverpod/riverpod.dart';
 
 final newsListProvider =
-    AsyncNotifierProvider.autoDispose<AsyncNewsListNotifier, List<NewsEntry>>(
+    AsyncNotifierProvider.family<AsyncNewsListNotifier, List<NewsEntry>,String?>(
```

Comment:
Any particular reason it isn't ```autoDispose``` too?

Reply:
Let me add it.

Reply:
Yes, because we are using it on space navbar (space_navbar_provider.dart) also and other there it will give compile time error. All the other providers are also used in this manner.